tests/test_selenium.py

```diff
@@ -0,0 +1,144 @@
+from page_objects.electric_bikes import ElectricBikesPage
+from page_objects.main_page import MainPage
+from page_objects.shops_page import ShopsPage
+from page_objects.sale_page import SalePage
+from utilities.config_reader import ReadConfig
+from selenium.webdriver.support.ui import Select
+import pytest
+
+
+@pytest.mark.smoke
+def test_title(create_driver):
+    driver = create_driver
+    actual_title = driver.title
+    assert actual_title == "Trek Bikes - The world's best bikes and cycling gear"
+
+
+@pytest.mark.smoke
+@pytest.mark.regression
+def test_login(create_driver):
+    user_name, password = ReadConfig.get_user_creds()
+    driver = create_driver
+    MainPage(driver).click_person_icon_button().click_log_in_button().set_email(user_name).set_password(
+        password).click_log_in_button()
+
+
+@pytest.mark.smoke
+def test_selection_region(create_driver):
+    driver = create_driver
+    main_page = MainPage(driver).open_region_selection_page().select_netherlands_region()
+    assert main_page.check_if_netherlands_region_is_applied(), 'Smth is wrong, Netherlands regions is not applied'
+
+
+@pytest.mark.regression
+@pytest.mark.parametrize("navigation_button", ["mountain", "road", "gravel", "shops", "sale", "electric_bikes"])
+def test_redirect_to_navigation_blocks(create_driver, navigation_button):
+    driver = create_driver
+    main_page = MainPage(driver)
+
+    if navigation_button == "mountain":
+        main_page.check_mountain_bikes_block()
+    elif navigation_button == "road":
+        main_page.check_road_bikes_block()
+    elif navigation_button == "gravel":
+        main_page.check_gravel_bikes_block()
+    elif navigation_button == "shops":
+        main_page.click_shops_button()
+        assert ShopsPage(driver).check_if_shops_is_loaded(), "shop page is not displayed normally"
+    elif navigation_button == "sale":
+        main_page.click_sale_button()
+        assert SalePage(driver).check_if_sales_page_is_opened(), "sale page is not displayed properly"
+    elif navigation_button == "electric_bikes":
+        main_page.open_electric_bikes()
+        assert ElectricBikesPage(driver).check_electric_bikes_page(), "electric bikes page is not opened properly"
```

Comment:
redesign test. it is not a good idea to have so many if statements in the test. Better not to have it at all